2.0/m_antirandom.cpp

```diff
@@ -550,9 +568,9 @@ class ModuleAntiRandom : public Module
 		unsigned int digits = 0;
 
 		/* Fast digit/consonant/vowel checks... */
-		for (s = original_str; *s; s++)
+		for (const char &c : original_str)
```

Comment:
InspIRCd is not built with -std=c++11 and on some compilers there may be an ABI difference between -std=c++03 and -std=c++11. Can you change this to a pre-C++11 for loop?